providers/implementations/keymgmt/ml_kem_kmgmt.c

```diff
@@ -81,6 +82,10 @@ static int ml_kem_pairwise_test(const ML_KEM_KEY *key)
         return 1;
 
 #ifdef FIPS_MODULE
+    /* During self test, it is a waste to do this test */
+    if (ossl_fips_self_testing())
+        return 1;
+
```

Comment:
Should be harmless, but presently unnecessary. If you're not *planning* to add tests involving key import to the FIPS POST, this can be deleted and avoids the call and branch.

Reply:
This test is run on key generation too.  It's that case for which I added it.

Reply:
So the test could be made before the PCT is called in `ml_kem_gen()`, but this is fine too.